pkgs/development/libraries/plplot/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchzip, cmake }:
+
+stdenv.mkDerivation rec {
+  pname   = "plplot";
+  version = "5.15.0";
+
+  src = fetchzip {
```

Comment:
Oh and why is this `fetchzip` and not `fetchurl`?

Reply:
I wasn't sure if SourceForge archives are unstable in the way that Github ones are.

Reply:
Well if `fetchurl` works please switch to it.

Reply:
But how would I know if the archives are sporadically regenerated?

Reply:
So you are not sure if the download itself is reproducible? You can't know either way, since the download is cached in the store anyway. You _can_ be somewhat sure if you'll garbage collect your /nix/store and then change to `fetchurl` and then retest the build - in order to avoid cache use because of the hash staying the same.

Reply:
`fetchurl` seems more common in SourceForge derivations so I'll switch.

Reply:
Done.